config/config.go

```diff
@@ -147,21 +146,15 @@ func Load(
 		}
 	}
 
-	// Start with extensions and service.
+	// Start with the service extensions.
 
 	extensions, err := loadExtensions(v, factories.Extensions)
 	if err != nil {
 		return nil, err
 	}
 	config.Extensions = extensions
 
-	service, err := loadService(v)
-	if err != nil {
-		return nil, err
-	}
-	config.Service = service
-
-	// Load data components (receivers, exporters, processores, and pipelines).
+	// Load data components (receivers, exporters, and processores).
```

Comment:
typo of processors.

Reply:
eagle eye! I will fix before merging it.